cli/src/main/scala/com/lightbend/rp/reactivecli/runtime/kubernetes/Deployment.scala

```diff
@@ -127,6 +139,24 @@ object Deployment {
         }
         .toMap
 
+    private[kubernetes] def mergeEnvs(envs: Map[String, EnvironmentVariable]*): Map[String, EnvironmentVariable] = {
+      val concatLiteral = Set("RP_JAVA_OPTS")
```

Comment:
Just a minor thing, but I'd put `concatLiteral` as a constant in the `Deployment. RpEnvironmentVariables` object called `concatLiteralEnvironmentVariables`